pkg/project/expander_test.go

```diff
@@ -182,13 +171,9 @@ func TestExpandProjectSecret(t *testing.T) {
 func TestExpandProjectAlternateSyntax(t *testing.T) {
 	prj := loadProject(t)
 
-	expanded, err := project.ExpandFromProject("${platform.os}", prj)
+	expanded, err := project.ExpandFromProject("${project.name()}", prj)
 	assert.NoError(t, err, "Ran without failure")
-	if runtime.GOOS != "darwin" {
-		assert.Equal(t, runtime.GOOS, expanded, "Expanded platform variable")
-	} else {
-		assert.Equal(t, "macos", expanded, "Expanded platform variable")
-	}
+	assert.Equal(t, "general", expanded, "Expanded platform variable")
```

Comment:
This error string should be updated as we are no longer expanding the platform variable

Reply:
Thanks for catching this.